app/apollo/resolvers/channel.js

```diff
@@ -489,7 +493,10 @@ const channelResolvers = {
         if(!channel){
           throw new NotFoundError(context.req.t('Channel uuid "{{channel_uuid}}" not found.', {'channel_uuid':uuid}), context);
         }
+
+        logger.info({req_id, user, org_id, uuid, name}, `${queryName} validating`);
```

Comment:
This log line shouldn't be moved from its prior location just after the `try{` line -- the lines between there and here are also 'validation'.